server/master/src/main/java/org/apache/accumulo/master/TabletGroupWatcher.java

```diff
@@ -204,7 +204,8 @@ public void run() {
           }
           Master.log.debug(store.name() + " location State: " + tls);
           // ignore entries for tables that do not exist in zookeeper
-          if (TableManager.getInstance().getTableState(tls.extent.getTableId()) == null)
+          TableState tableState = TableManager.getInstance().getTableState(tls.extent.getTableId());
+          if (tableState == null || tableState == TableState.DELETING)
```

Comment:
Thinking about this some more. Ingoring deleting tablets may prevent unloading them, which would cause delete table to hang.